README.md

```diff
@@ -43,7 +43,7 @@ The awx.awx or ansible.controller collections MUST be installed in order for thi
 
 |                                      Collection Name                                       |                      Purpose                      |
 |:------------------------------------------------------------------------------------------:|:-------------------------------------------------:|
-| [AAP Configuration Extended](https://github.com/redhat-cop/aap_configuration_extended)     | Where other useful roles that don't fit here live |
+| [AAP Configuration](https://github.com/redhat-cop/infra.aap_configuration)                 | Easy interaction with an AWX or Ansible Automation Platform via Ansible roles using the AWX/Controller collection modules |
```

Comment:
I would just say something like "Easy interaction with Ansible Automation Platform via Ansible roles"

Reply:
done in [new commit](https://github.com/redhat-cop/aap_configuration_extended/pull/25/commits/b8c4cdb0d26e5ea76648aa840f781aaa13f46813)